### ISSUE TITLE
generate synthetic data

### ISSUE THREAD

**Issue by blacksnail789521** [davidhallac/TICC#48]
Hi, 

I want to ask about how do yo generate synthetic data based on previous samples?
In your paper, you mentioned that "The data is then drawn one sample at a time, conditioned on the values of the previous w − 1 samples.".
I know you generate w samples (n * w array) using "numpy.random.multinomial", but how do you generate w samples based on previous w − 1 samples?
It means how do I generate 1 sample given w - 1 samples?

Thank you for your time.

**Comment by davidhallac** [comment]
Hi,

Please look at TICC/paper code/generate_synthetic_data.py for code to generate the data.

Thanks!

**Comment by blacksnail789521** [comment]
Hi, 
I use your code to generate the synthetic data, but now I have a problem.
When I generate one single data subsequence (w samples), it shows that the later number is bigger (ex: 1, 3, 9).
So if I generate another samples based on previous w - 1 samples, this sample will becomes very large (ex: based on 3, 9, now I will generate 27).
If I set #timestamps very large, the synthetic data will continue to grow until nan appears.
Do you know how to solve this problem?
Thank you for your time.

**Comment by davidhallac** [comment]
Hmm, what parameters did you use when generating it? Did you change anything from the defaults?

**Comment by Mehakkhan7** [comment]
i haven't implemented anything yet, thanks
________________________________

**Comment by blacksnail789521** [comment]
Hi, 

Now I use your original code, and it's fine, problem-free.
I used to feed my own adjacency matrix into your code.
This is the adjacency matrix I fed into:
![image](https://user-images.githubusercontent.com/5949636/45262178-64664580-b444-11e8-8eb8-35584f187654.png)
I create theta based on your rule. (Let c be the smallest eigenvalue of G, and set Θi = G+(0.1+|c |)I.)
Maybe it's because I set the diagonal terms to zeros, I don't know.

Thank you very much for your help.
